react/features/whiteboard/middleware.ts

```diff
@@ -58,12 +58,21 @@ MiddlewareRegistry.register((store: IStore) => (next: Function) => async (action
         if (!existingCollabDetails) {
             const collabDetails = await generateCollaborationLinkData();
             const collabServerUrl = getCollabServerUrl(state);
+            const roomId = getCurrentRoomId(state);
 
             focusWhiteboard(store);
-            dispatch(setupWhiteboard({ collabDetails }));
+            dispatch(setupWhiteboard({
+                collabDetails: {
```

Comment:
I'd create a `collabDetails` object which is then passed to both, avoiding a potential de-sync.